uwuw/uwuw_preprocessor.cpp

```diff
@@ -0,0 +1,693 @@
+#include "uwuw_preprocessor.hpp"
+
+#define DAG moab::DagMC::instance()
+
+// constructor
+uwuw_preprocessor::uwuw_preprocessor(std::string material_library_filename, std::string dagmc_filename,
+                                     std::string output_file,  bool verbosity, bool fatal_errors)
+{
+  // make new name concatenator class
+  ncr = new name_concatenator();
+
+  // load the materials
+  material_library = mat_lib.load_pyne_materials(material_library_filename,"/material_library/materials");
+
+  // load the material objects
+  // load the dag file
+  moab::ErrorCode rval = DAG->load_file(dagmc_filename.c_str());
+
+  // do the minimal DAGMC initialisation
+  rval = DAG->setup_impl_compl();
+  rval = DAG->setup_indices();
+
+  // set the output filename
+  output_filename = output_file;
+
+  // set the verbosity
+  verbose = verbosity;
+  fatal = fatal_errors;
+}
+
+// destructor
+uwuw_preprocessor::~uwuw_preprocessor()
+{
+}
+
+// get the dagmc properties from the geometry file
+std::map<moab::EntityHandle, std::vector<std::string> > get_property_assignments( std::string property,
+    int dimension,
```

Comment:
I don't know how much style matters here, but this indentation is inconsistant with the indentation of the constuctor.